XrmToolBox/Forms/PluginsChecker.cs

```diff
@@ -141,6 +83,84 @@ public void RefreshPluginsList()
             bw.RunWorkerAsync();
         }
 
+        private ListViewItem AddPackage(IPackage package)
+        {
+            var xtbPackage = new XtbNuGetPackage(package, PackageInstallAction.None);
+            var item = new ListViewItem(xtbPackage.ToString());
+            item.Tag = xtbPackage;
+            item.SubItems.Add(package.Version.ToString());
+            var currentVerItem = item.SubItems.Add("");  //Current version
+            item.SubItems.Add(package.Description);
+            item.SubItems.Add(string.Join(", ", package.Authors));
+            var actionItem = item.SubItems.Add("None");
+
+            var files = package.GetFiles();
+
+            bool install = false, update = false, compatible = false;
+
+            var xtbDependency = package.FindDependency("XrmToolBox", null);
+            if (xtbDependency != null)
+            {
+                var xtbDependencyVersion = xtbDependency.VersionSpec.MinVersion.Version;
+                compatible = IsPluginDependencyCompatible(xtbDependencyVersion);
+            }
+
+            var currentVersion = new Version(int.MaxValue, int.MaxValue, int.MaxValue, int.MaxValue);
+            var currentVersionFound = false;
+
+            // TODO: Don't compare with all files, plugin packages may contain other dll's and exe's that have other versioning

```

Comment:
Regardless if the column is there, it is a problem when e.g. this package: https://www.nuget.org/packages/DLaB.Xrm.XrmToolBoxTools/ states the available version is 1.2016.3.31, and all files in the package will be compared to that version. His package contains files that follow different versioning, which is out of his control. E.g. Newtonsoft.Json.dll that has version 6.0.5.xxxx.
The problem is that **all** files are compared, and if any of these have a lower version than the package it will encourage update. And the same will happen next time, even if it is the same package.